packages/web-components/src/components/background-media/background-media.ts

```diff
@@ -122,40 +119,36 @@ class C4DBackgroundMedia extends C4DImage {
     );
     const assignedVideos = assignedElements.filter(
       (el) => el.tagName === `${c4dPrefix}-video-player-container`.toUpperCase()
-    );
+    ) as C4DVideoPlayerContainer[];
 
-    if (
-      assignedElements.length === assignedImages.length &&
-      !assignedVideos.length
-    ) {
+    if (assignedImages.length && !assignedVideos.length) {
       this.containsOnlyImages = true;
     }
-
-    if (assignedVideos.length) {
-      const [video] = assignedVideos;
-      this.videoId = video.getAttribute('video-id');
-      this.videoPlayer = video.querySelector(`${c4dPrefix}-video-player`);
-      this.videoIsPlaying = (video as C4DVideoPlayerContainer).isPlaying;
```

Comment:
Can we remove the `videoIsPlaying` property altogether from this component? Doesn't look like its used anymore.